test/navigationLibrary/StandardRoute/StandardRouteModel.spec.js

```diff
@@ -3,6 +3,7 @@ import sinon from 'sinon';
 import _isEqual from 'lodash/isEqual';
 import _keys from 'lodash/keys';
 import _map from 'lodash/map';
+import _uniq from 'lodash/uniq';
 
 import StandardRouteModel from '../../../src/assets/scripts/client/navigationLibrary/StandardRoute/StandardRouteModel';
```

Comment:
as mentioned above, we should add a test for cases where one of the collections is `undefined`.

for setup, you could do:
```javascript
const standardRouteModel = new StandardRouteModel(STAR_LIST_MOCK.GRNPA1);
standardRouteModel._exitCollection = undefined;
```
then run `standardRouteModel.getAllFixNamesInUse()` and see if stuff blows up.